src/main/java/ToDo.java

```diff
@@ -0,0 +1,18 @@
+public class ToDo extends Task {
+
+    public ToDo(String description) {
+        super(description);
+    }
```

Comment:
another thing is maybe you can write your commit more detailed such as "level-0 completed" this will help revierer to nowyour progress